src/build/common/steps/nuget-license-check.steps.yaml

```diff
@@ -0,0 +1,39 @@
+# Steps to run nuget license checks 
+parameters:
+  - name: projectPaths
+    type: object
+    default: []
+
+steps:
+  - script: |
+      dotnet new tool-manifest
+      dotnet tool install dotnet-project-licenses
```

Comment:
Is this the one being installed? It says it is abandoned

https://github.com/tomchavakis/nuget-license?tab=readme-ov-file#:~:text=This%20project%20seems%20to%20be%20abandoned